src/util.ts

```diff
@@ -331,7 +348,7 @@ class Logger {
      * @param msg Message to write
      * @returns {void}
      */
-    debug(module: string, msg: string): void {
+    debug(module: string, msg: unknown): void {
```

Comment:
Please change this upsteam instead.
The logging function should not need to handle non-strings.

Reply:
Thank you for the comment. I changed its type to unknown because we're passing it directly to this.log function
![image](https://github.com/user-attachments/assets/ad36fb7f-6f8a-45c5-9c79-396b35dbed20)

where its type is given as unknown.
![image](https://github.com/user-attachments/assets/91e6d551-3221-468f-bbbf-b7c53c5dfb64)

We are giving it a non-string msg to log
![image](https://github.com/user-attachments/assets/54fae9f5-2b75-4fd6-aa54-220adf26f364)

Here it was giving typescript error so I changed it to unknow.
Having string type was giving typescript error that I have added in screenshots

Please let me know if this is incorrect approach or if there is any other way

Reply:
I meant that `this.hideLog` should be changed, instead of `debug`

Reply:
Ok yes, changing ```this.hidelog``` with ```json.stringify``` of it would be correct.

But louislam has already done a change at this location so I'll revert this change